README.md

```diff
@@ -177,14 +177,14 @@ A chai [should](https://www.chaijs.com/api/bdd/) instance, containing the `bignu
 
 ---
 
-### shouldFail
-Collection of assertions for failures (similar to [chai's `throw`](https://www.chaijs.com/api/bdd/#method_throw)). `shouldFail` will accept any exception type, but more specific functions exist and their usage is encouraged.
+### expectFailure
```

Comment:
To be honest I also dislike the name, but instead of trying to come up with a better one I simply went through and expected your review. :upside_down_face:

I really like your proposal.